### DISCUSSION TITLE
Ability to Restrict Access to Different Charts under the Same Datasource

### DISCUSSION THREAD

**Discussion by lihanlilyy** [apache/superset#22732] [Q&A / Help]
Hi, just wondering if it is possible to restrict access to charts created using the same datasource to different user groups through row level access. 

For example, I used a datasource named "DS" to create charts "Chart A" and "Chart B". There are 2 user groups "User A" and "User B". Is it possible to restrict the settings such that:
- only User A can view Chart A, and
- only User B can view Chart B?

Thank you.

**Comment by victorlamello** [comment]
I'm on the same path here... there is still no solution for this issue?
